test/hotspot/jtreg/compiler/loopopts/superword/SumRedAbsNeg_Double.java

```diff
@@ -110,6 +91,9 @@ public static void sumReductionInit(
         }
     }
 
+    @Test
+    @IR(applyIf = {"SuperWordReductions", "false"},
+        failOn = {IRNode.ADD_REDUCTION_VD})
```

Comment:
This might fail with `sve` support. But not sure if it is implemented there.
Also, this test only runs with `@requires os.arch=="aarch64" | os.arch=="riscv64"`.
If anything, this test should check for `AbsD` and `NegD`. Would be interesting to check if this is implemented on intel architecture by now? If yes, change the requirement and add a corresponding IR rule.